modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -14,6 +14,51 @@ include::hello-world:partial$installation.adoc[]
+| Checksums                     | Any | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5.sha256sum[libcouchbase-3.0.5.sha256sum]
+| Source Archive                | Any | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5.tar.gz[libcouchbase-3.0.5.tar.gz]
+| Amazon Linux 2                | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_amzn2_x86_64.tar[libcouchbase-3.0.5_amzn2_x86_64.tar]
+| Enterprise Linux 7            | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_centos7_x86_64.tar[libcouchbase-3.0.5_centos7_x86_64.tar]
+| Enterprise Linux 8            | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_centos8_x86_64.tar[libcouchbase-3.0.5_centos8_x86_64.tar]
+| Ubuntu 16.04 (xenial)         | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_ubuntu1604_xenial_amd64.tar[libcouchbase-3.0.5_ubuntu1604_xenial_amd64.tar]
+| Ubuntu 18.04 (bionic)         | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_ubuntu1804_bionic_amd64.tar[libcouchbase-3.0.5_ubuntu1804_bionic_amd64.tar]
+| Ubuntu 20.04 (focal)          | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_ubuntu2004_focal_amd64.tar[libcouchbase-3.0.5_ubuntu2004_focal_amd64.tar]
+| Debian 9 (stretch)            | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_debian9_stretch_amd64.tar[libcouchbase-3.0.5_debian9_stretch_amd64.tar]
+| Debian 10 (buster)            | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_debian10_buster_amd64.tar[libcouchbase-3.0.5_debian10_buster_amd64.tar]
+| Visual Studio 2015 (VC14)     | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_vc14_amd64.zip[libcouchbase-3.0.5_vc14_amd64.zip]
+| Visual Studio 2017 (VC15)     | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_vc15_amd64.zip[libcouchbase-3.0.5_vc15_amd64.zip]
+| Visual Studio 2015 TLS (VC14) | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_vc14_amd64_openssl.zip[libcouchbase-3.0.5_vc14_amd64_openssl.zip]
+| Visual Studio 2017 TLS (VC15) | x64 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_vc15_amd64_openssl.zip[libcouchbase-3.0.5_vc15_amd64_openssl.zip]
+// Hidden as we might not want to encourage people using 32-bit platforms
+// | Visual Studio 2015 (VC14)     | x32 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_vc14_x86.zip[libcouchbase-3.0.5_vc14_x86.zip]
+// | Visual Studio 2017 (VC15)     | x32 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_vc15_x86.zip[libcouchbase-3.0.5_vc15_x86.zip]
+// | Visual Studio 2015 TLS (VC14) | x32 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_vc14_x86_openssl.zip[libcouchbase-3.0.5_vc14_x86_openssl.zip]
+// | Visual Studio 2017 TLS (VC15) | x32 | https://packages.couchbase.com/clients/c/libcouchbase-3.0.5_vc15_x86_openssl.zip[libcouchbase-3.0.5_vc15_x86_openssl.zip]
```

Comment:
@ingenthr should we uncomment these lines, or just have all packages deployed on S3, but not include them into release notes?

Reply:
I'm going to merge commented out for now - can be changed later if @ingenthr wishes.